### PR TITLE
Update Best Buy, uses 2FA

### LINKED ISSUE #4250: Best Buy uses 2FA
https://www.bestbuy.com/site/help-topics/2-step-verification/pcmcat1561056149844.c?id=pcmcat1561056149844

### PR DESCRIPTION
**Pull request by bby8x7crhl6g**
Fixes #4250

### CODE REVIEW

**Comment by Rfosu2k2** [comment]
It appears that since I added to this site, Best Buy has now enabled 2FA with creating the user on github and submitting the change @here!  Thanks twofactorauth!

File: _data/retail.yml [hunk 1]
```diff
@@ -53,8 +53,10 @@ websites:
   - name: Best Buy
     url: https://www.bestbuy.com/
     img: bestbuy.png
-    twitter: BestBuySupport
-    facebook: bestbuy
+    tfa:
+      - sms
+      - software
```
Comment (Carlgo11) [inline _data/retail.yml]:
```suggestion
      - totp
```

File: _data/retail.yml [hunk 2]
```diff
@@ -53,8 +53,10 @@ websites:
...
+    doc: https://www.bestbuy.com/site/help-topics/2-step-verification/pcmcat1561056149844.c?id=pcmcat1561056149844
```
Comment (Carlgo11) [inline _data/retail.yml]:
```suggestion
    doc: https://www.bestbuy.com/site/help-topics/2-step-verification/pcmcat1561056149844.c
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: _data/retail.yml (hunk 2 above)
Comment (bby8x7crhl6g) [reply _data/retail.yml]: While both the URLs, with and without the id query param, result in the same page being displayed, the canonical link is with the id query param.  The canonical URL is the preferred version. 

`<link rel="canonical" href="https://www.bestbuy.com/site/help-topics/2-step-verification/pcmcat1561056149844.c?id=pcmcat1561056149844">`
https://en.wikipedia.org/wiki/Canonical_link_element

Comment (Carlgo11) [reply _data/retail.yml]: We want to keep the URLs as short as possible so please remove any part of the URL that's not required for the page to load.

**Review by kmpoppe — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**